src/Program.fs

```diff
@@ -15,8 +15,11 @@ let [<Literal>] FableRemotingJsonVersion = "2.17.0"
 let [<Literal>] FableSimpleHttpVersion = "3.0.0"
 let [<Literal>] FableSimpleJsonVersion = "3.21.0"
 let [<Literal>] FSharpCoreVersion = "4.7.2"
+let [<Literal>] FSharpSystemTextJsonVersion = "0.17.4"
+let [<Literal>] FSharpControlFusionTasks = "2.2.0"
```

Comment:
Please exclude the Control fusion tasks from here, consumers can add it if they want